src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -186,46 +175,52 @@ def get_display(my):
         files_div.add_style("max-height: 300px")
         files_div.add_style("height: 300px")
         files_div.add_style("overflow-y: auto")
-        files_div.add_border()
         files_div.add_style("padding: 3px")
-        files_div.add_color("background", "background3")
+        files_div.add_color("background", background_mouseout)
+        files_div.add_style("border: 3px dashed %s" % border_color_light)
+        files_div.add_style("border-radius: 20px 20px 20px 20px")
+        files_div.add_style("z-index: 1")
         #files_div.add_style("display: none")
 
         bgcolor = div.get_color("background3")
         bgcolor2 = div.get_color("background3", -3)
 
+        #style_text = "text-align: center; margin-top: 100px; color: #A0A0A0; font-size: 3.0em; z-index: 10;"
+
+        background = DivWdg()
+        background.add_class("spt_files_background")
+        files_div.add(background)
+
+        background.add_style("text-align: center")
+        background.add_style("margin-top: 100px")
+        background.add_style("opacity: 0.65")
+        background.add_style("font-size: 3.0em")
+        background.add_style("z-index: 10")
+
+        background_text = DivWdg("<p>Drag Files Here</p>")
+
+        background.add(background_text)
+
         files_div.add_behavior( {
-            'type': 'mouseenter',
+            'type': 'mouseover',
             'bgcolor': bgcolor2,
             'cbjs_action': '''
-            bvr.src_el.setStyle("background", bvr.bgcolor)
-            '''
+            bvr.src_el.setStyle("border","3px dashed %s")
+            bvr.src_el.setStyle("background","%s")
+            ''' % (border_color_dark, background_mouseenter)
         } )
 
```

Comment:
Mouseover is used rather than mouse enter because using mouseenter causes an ui bug, which when mousing over the "Drag Files Here" background text in the ingest wdg, the other parts of the background lose their coloring.